examples/azure/terraform-azure-mysqldb-example/main.tf

```diff
@@ -30,8 +30,8 @@ resource "azurerm_resource_group" "mysql_rg" {
 # Random password is used as an example to simplify the deployment and improve the security of the database.
 # This is not as a production recommendation as the password is stored in the Terraform state file.
 resource "random_password" "password" {
-  length = 16
-  special = true
+  length           = 16
+  special          = true
```

Comment:
Ah sorry I caused hassle here. I just merged the fix for this in master so you might have conflicts with it.

Reply:
no probs - I'll merge!